twitter-eventlistener-plugin/src/main/java/com/twitter/presto/plugin/eventlistener/QueryCompletedEventScriber.java

```diff
@@ -55,6 +60,23 @@ public void handle(QueryCompletedEvent event)
     }
   }
 
+  private static long getLongOrZero(String strVal)
+  {
+    try {
+      return Long.valueOf(strVal).longValue();
```

Comment:
The ``strVal`` appears here should be a string representation of io.airlift.units.DataSize